CHANGES.txt

```diff
@@ -150,6 +150,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Fix another instance of `int main()` in CheckLib() causing failures
       when using -Wstrict-prototypes.
 
+  From Vitaly Cheptsov:
+    - Fix race condition in `Mkdir` which can happen when two `SConscript`
+      are parsed simultaneously by two separate build commands.
```

Comment:
change 'parsed' to 'processed' ?

Reply:
Sure, changed here and in the code comment as well.